ir/src/main/scala/org/scalajs/core/ir/Printers.scala

```diff
@@ -681,10 +681,19 @@ object Printers {
 
         case IntLiteral(value) =>
           if (value >= 0) {
-            print(value)
+            print(value.toString)
           } else {
             print('(')
-            print(value)
+            print(value.toString)
+            print(')')
+          }
+
+        case LongLiteral(value) =>
+          if (value >= 0) {
+            print(value.toString)
+          } else {
+            print('(')
+            print(value.toString)
             print(')')
```

Comment:
Oh, yes. Probably. I mean that the end result should show an "L". No matter how you achieve it :)
(probably just adding `print('L')` is easiest).
It might just be nice to see the distinction between an int literal and a long literal. Just like in the IR we also print things like `+[int]`.